app/pages/sections/order-items/associated-services/select/routes.js

```diff
@@ -60,5 +64,27 @@ export const associatedServicesSelectRoutes = (authProvider, addContext, session
     return res.render('pages/sections/order-items/associated-services/select/price/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.post('/associated-service/price', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
+    const { orderId } = req.params;
+
+    const response = validateAssociatedServicePriceForm({ data: req.body });
+    if (response.success) {
+      sessionManager.saveToSession({ req, key: 'selectedPriceId', value: req.body.selectAssociatedServicePrice });
+      logger.info('redirecting to associated services select recipient page');
+      return res.redirect(`${config.baseUrl}/organisation/${orderId}/associated-services/select/associated-service/price/recipient`);
```

Comment:
this needs to change to 
```
return res.redirect(`${config.baseUrl}/organisation/${orderId}/associated-services/neworderitem`);
```

There is no recipient select page for associated service